### PR TITLE
Updates docs with "Add Text to Existing Image" examples

### PR DESCRIPTION
**Pull request by 1kevgriff**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/dlemstra/Magick.NET/pulls) open
- [x] I have verified that I am following the existing coding patterns and practices as demonstrated in the repository.

### Description

Added an item to the documentation for adding text to an existing image.

Added sample code to examples solution for adding text to an existing image.

### CODE REVIEW

File: docs/Drawing.md [hunk 1]
```diff
@@ -19,4 +19,37 @@ using (var image = new MagickImage(new MagickColor("#ff00ff"), 512, 128))
...
+## Adding Text To Existing Image
+
+![Example of adding text to existing image](./img/example_addTextToExistingImage.jpg)
+
+```C#
+var pathToBackgroundImage = "path/to/background.png";
+var pathToNewImage = "path/to/newImage.png";
+var textToWrite = "Insert This Text Into Image";
+
+/* These settings will create a new caption
```
Comment (dlemstra) [inline docs/Drawing.md:33]: Could you change the `/*` to `//` for each line to make sure it is in the rest of the style of the repo?

File: docs/Drawing.md [hunk 2]
```diff
@@ -19,4 +19,37 @@ using (var image = new MagickImage(new MagickColor("#ff00ff"), 512, 128))
...
+var readSettings = new MagickReadSettings()
+{
+    Font = "Calibri",
+    TextGravity = Gravity.Center,
+    BackgroundColor = MagickColors.Transparent,
+    Height = 250, // height of text box
+    Width = 680 // width of text box
+};
+
+using (var image = new MagickImage(pathToBackgroundImage))
```
Comment (dlemstra) [inline docs/Drawing.md:46]: Could you not "layer" the usings and add braces?

File: samples/Magick.NET.Samples/Drawing.cs [hunk 3]
```diff
@@ -35,5 +37,36 @@ public static void DrawText()
                   .Draw(image);
             }
         }
+
+        public static void AddTextToExistingImage()
+        {
+            var pathToBackgroundImage = Path.Combine(Environment.CurrentDirectory, "Files/2FD-Background.jpg");
+
```
Comment (dlemstra) [inline samples/Magick.NET.Samples/Drawing.cs:44]: There is an extra new line here.

File: samples/Magick.NET.Samples/Drawing.cs [hunk 4]
```diff
@@ -35,5 +37,36 @@ public static void DrawText()
                   .Draw(image);
             }
         }
+
+        public static void AddTextToExistingImage()
+        {
+            var pathToBackgroundImage = Path.Combine(Environment.CurrentDirectory, "Files/2FD-Background.jpg");
```
Comment (dlemstra) [inline samples/Magick.NET.Samples/Drawing.cs:43]: Could you use the `Files` class? And add the images to the repo?

File: docs/Drawing.md [hunk 5]
```diff
@@ -19,4 +19,37 @@ using (var image = new MagickImage(new MagickColor("#ff00ff"), 512, 128))
       .Ellipse(256, 96, 192, 8, 0, 360)
       .Draw(image);
 }
-```
\ No newline at end of file
+```
+
+## Adding Text To Existing Image
+
+![Example of adding text to existing image](./img/example_addTextToExistingImage.jpg)
```
Comment (dlemstra) [inline docs/Drawing.md:26]: This could be the image in the sample directory.

File: docs/Drawing.md (hunk 1 above)
Comment (1kevgriff) [reply docs/Drawing.md:33]: 👍

File: samples/Magick.NET.Samples/Drawing.cs (hunk 3 above)
Comment (1kevgriff) [reply samples/Magick.NET.Samples/Drawing.cs:44]: 👍

File: samples/Magick.NET.Samples/Drawing.cs (hunk 4 above)
Comment (1kevgriff) [reply samples/Magick.NET.Samples/Drawing.cs:43]: 👍

File: docs/Drawing.md (hunk 5 above)
Comment (1kevgriff) [reply docs/Drawing.md:26]: 👍

File: docs/Drawing.md (hunk 2 above)
Comment (1kevgriff) [reply docs/Drawing.md:46]: 👍

**Pull request merged**